### PR TITLE
[VPP] black tiled video wall issue

### PR DESCRIPTION
**Pull request by IlyaSergeyev**
Issue #803 

Driver should not use colorfill for tiled video wall (works on APL)

Upd:
        m_pipelineParam[i].pipeline_flags |= VA_PROC_PIPELINE_SUBPICTURES;
        m_pipelineParam[i].filter_flags |= VA_FILTER_SCALING_HQ;

works for both: APL, CFL and SKL

### CODE REVIEW

File: _studio/shared/src/mfx_vpp_vaapi.cpp [hunk 1]
```diff
@@ -1994,12 +1999,14 @@ mfxStatus VAAPIVideoProcessing::Execute_Composition(mfxExecuteParams *pParams)
         if (pParams->bComposite)
         {
             m_pipelineParam[refIdx].num_filters  = 0;
-#if defined(LINUX_TARGET_PLATFORM_BXT) || defined(LINUX_TARGET_PLATFORM_BXTMIN)
-            m_pipelineParam[refIdx].pipeline_flags |= VA_PROC_PIPELINE_SUBPICTURES;
-            m_pipelineParam[refIdx].filter_flags   |= VA_FILTER_SCALING_HQ;
-#else
-            m_pipelineParam[refIdx].pipeline_flags  |= VA_PROC_PIPELINE_FAST;
-#endif
+            if(MFX_HW_APL == hwType) {
```
Comment (alariono) [inline _studio/shared/src/mfx_vpp_vaapi.cpp]: @IlyaSergeyev, make APL variant of pipeline properties default for all platforms

Comment (dvrogozh) [reply _studio/shared/src/mfx_vpp_vaapi.cpp]: @IlyaSergeyev : I am not sure why this is resolved? I still see APL check...

File: _studio/shared/src/mfx_vpp_vaapi.cpp [hunk 2]
```diff
@@ -1650,14 +1653,12 @@ mfxStatus VAAPIVideoProcessing::Execute_Composition_TiledVideoWall(mfxExecutePar
...
         m_pipelineParam[i].pipeline_flags |= VA_PROC_PIPELINE_SUBPICTURES;
-        m_pipelineParam[i].filter_flags   |= VA_FILTER_SCALING_HQ;
-#else
-        m_pipelineParam[i].pipeline_flags  |= VA_PROC_PIPELINE_FAST;
-#endif
+        m_pipelineParam[i].filter_flags |= VA_FILTER_SCALING_HQ;
+
         m_pipelineParam[i].filters      = 0;
         m_pipelineParam[i].num_filters  = 0;
+        m_pipelineParam[i].output_background_color &= 0x00FFFFFF;
```
Comment (dvrogozh) [inline _studio/shared/src/mfx_vpp_vaapi.cpp]: why `&=`? What you really try to program here? I have some feeling that idea is to program `m_pipelineParam[i].output_background_color=0`...

File: _studio/shared/src/mfx_vpp_vaapi.cpp [hunk 3]
```diff
@@ -1497,6 +1497,9 @@ mfxStatus VAAPIVideoProcessing::Execute_Composition_TiledVideoWall(mfxExecutePar
     MFX_CHECK_NULL_PTR1( pParams->pRefSurfaces );
     MFX_CHECK_NULL_PTR1( pParams->pRefSurfaces[0].hdl.first );
 
+    VAAPIVideoCORE* hwCore = dynamic_cast<VAAPIVideoCORE*>(m_core);
```
Comment (dvrogozh) [inline _studio/shared/src/mfx_vpp_vaapi.cpp]: You don't use this anywhere below.

**Pull request merged**